.github/workflows/auto-assign.yml

```diff
@@ -0,0 +1,59 @@
+name: Auto assign pull requests
+
+on:
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  update_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: tibdex/github-app-token@v1.6.0
+        id: generate-token
+        with:
+          app_id: ${{ secrets.METABASE_BOT_APP_ID }}
+          private_key: ${{ secrets.METABASE_BOT_APP_PRIVATE_KEY }}
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Assign author and their team. Returns true if team exists.
+        id: auto-assign
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ steps.generate-token.outputs.token }}
+          script: |
+            const fs = require('fs');
+            const teamConfig = JSON.parse(fs.readFileSync('.github/team.json', 'utf-8'));
```

Comment:
what if we get team members based on the github teams? e.g. through `GET /teams/:team_id/members`

Reply:
Good idea! At this moment it does not always match the reality though. For instance, Anton works for both QB and QP teams, and with the current config allows to pick one team he works with the most or even disable such behavior if there is no good default